lib/search.js

```diff
@@ -96,12 +88,13 @@ module.exports = function (client, value, from, size, effect, weights) {
                     },
                     script_score: {
                         lang: 'groovy',
-                        script: 'pow(' +
+                        script: '(doc["name.raw"].value.equalsIgnoreCase(name)) ? 1 : _score * pow(' +
                         'doc["score.detail.popularity"].value * weightPopularity + ' +
                         'doc["score.detail.quality"].value * weightQuality + ' +
                         'doc["score.detail.maintenance"].value * weightMaintenance, ' +
                         'effect)',
                         params: {
+                            name: value,
```

Comment:
Agree, will change. Have been in a fight with Groovy and when it worked I just stopped changing. ahah